mmv1/products/dlp/api.yaml

```diff
@@ -297,7 +297,7 @@ objects:
               properties:
                 - !ruby/object:Api::Type::NestedObject
                   name: 'saveFindings'
```

Comment:
If I'm understanding correctly, `actions` should have at least one of `saveFindings` or `pubSub`?

If so, can we add `at_least_one_of` like in https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/accesscontextmanager/api.yaml#L865

Reply:
Actually, (thanks for the pointer!), I think we actually want `exactly_one_of`, since only one action can be specified in one block. Done.